openedx/core/djangoapps/xblock/runtime/blockstore_runtime.py

```diff
@@ -33,6 +34,14 @@ class BlockstoreXBlockRuntime(XBlockRuntime):
     def parse_xml_file(self, fileobj, id_generator=None):
         raise NotImplementedError("Use parse_olx_file() instead")
 
+    def service(self, block, service_name):
+        """
+        Adding in a hacky shim to make sure we can load the TransitionalService. Is there a better place to inject it?
```

Comment:
I know you're not using this anymore, but for the record, yes:

https://github.com/edx/edx-platform/blob/657da20c6e75f2c2e6a8c6c6d3833f5aa45deb0e/openedx/core/djangoapps/xblock/runtime/runtime.py#L80-L82

or

https://github.com/edx/edx-platform/blob/657da20c6e75f2c2e6a8c6c6d3833f5aa45deb0e/openedx/core/djangoapps/xblock/runtime/runtime.py#L218-L236

or

https://github.com/edx/edx-platform/blob/657da20c6e75f2c2e6a8c6c6d3833f5aa45deb0e/openedx/core/djangoapps/xblock/runtime/runtime.py#L411-L420

depending on how costly the service is to initialize and whether it's scoped to one XBlock or not.